README.md

```diff
@@ -1,8 +1,38 @@
-[![Build Status](CI_BADGE_URL goes here)](REPO_URL goes here)
+[![Build Status](https://github.com/blyscuit/docs/actions/workflows/deploy_fly.yml/badge.svg)](https://github.com/blyscuit/rails-ic/tree/chore/3-deployment-automation)
 
 ## Introduction
 
-> *App introduction goes here ...*
+Rails Internal Certificate.
+
+
+## Get Started
+
+### Requirements
+
+- Install ruby and set your local ruby version to `3.0.1`
+- Install rails `7.0.1`
+- Install node `16.13.2` (For creating web application)
+
+> 📝 If running on Apple M1, to build docker image, please make sure to set platform to AMD64 by `export DOCKER_DEFAULT_PLATFORM=linux/amd64`
+
+### Use Script
+
+```sh
+make dev
+```
+
+### Manually
+
+```
+bundle install
+bin/docker-prepare
+bin/rails s
+```
```

Comment:
This is probable, not mandatory — make files are easy to ready if someone wants to know what commands are called behind the `make dev` command.

Also, when running only `rails s`, the background worker (with Sidekiq) will not work.

See the `Procfile` file. 


> Side note, I realize that our templates is quite complex with its nested call: Make dev => bin/dev => foreman start -f Procfile.dev => rails serve 🥵 
> I've added an issue for that https://github.com/nimblehq/rails-templates/issues/410

Reply:
Thank you very much. This we be good for future features.